channels/applicationinsights-channel-js/src/Offline.ts

```diff
@@ -1,92 +1,104 @@
-import { getWindow, getDocument, getNavigator, isUndefined, isNullOrUndefined, attachEvent } from "@microsoft/applicationinsights-core-js";
-import dynamicProto from "@microsoft/dynamicproto-js";
+import { getWindow, getDocument, getNavigator, isUndefined, isNullOrUndefined, createUniqueNamespace, mergeEvtNamespace, eventOn, eventOff } from "@microsoft/applicationinsights-core-js";
+
```

Comment:
The refactoring here is mostly just removing the internal Offline class (for minufication purposes), the functionality does not change.